frontend/__tests__/routes/_protected+/personal-information+/phone-number+/edit.test.tsx

```diff
@@ -67,7 +70,6 @@ describe('_gcweb-app.personal-information.phone-number.edit', () => {
 
       expect(data).toMatchObject({
         meta: {},
-        userInfo: { id: 'some-id', phoneNumber: '(111) 222-3333' },
       });
```

Comment:
This unit test passes but does not perform the correct assertion. The user's phone number should be found and returned as part of the `loader()`.

Reply:
i changed the expected data.